bedrock/settings/base.py

```diff
@@ -1326,6 +1336,22 @@ def JINJA_CONFIG():
         ),
         'output_filename': 'js/firefox_os-bundle.js',
     },
+    'firefox_os_new': {
+        'source_filenames': (
+            'js/base/mozilla-modal.js',
+            'js/libs/jquery.waypoints.min.js',
+            'js/libs/jquery.waypoints-sticky.min.js',
+            'js/libs/tweenmax.1.9.7.min.js',
```

Comment:
I think there's various files here from the old page that may not be needed?

* 'js/libs/tweenmax.1.9.7.min.js',
* 'js/libs/superscrollorama-1.0.1.js',
* js/libs/jquery.color.js',
* 'js/firefox/os/desktop.js',
* 'js/firefox/os/have-it.js',